inc/ajax.php

```diff
@@ -17,6 +17,11 @@
 function sell_media_add_to_cart() {
 	global $sm_cart;
 
+	if(!isset($_POST['_wpnonce']) || !isset($_POST['_wpnonce']) && !wp_verify_nonce($_POST['_wpnonce'], 'sell_media_add_cart_action')) {
```

Comment:
Is this line correct? checking same thing 2 times?

Reply:
@maharzan  I have updated code.